kueyen/metallb/ippool.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
   namespace: metallb-system
 spec:
   addresses:
-  - 139.229.144.50-139.229.144.100
```

Comment:
This IP address range needs to be de-allocated in ipam.

Reply:
yes, all the IPs are now updated 👍🏻 
<img width="1129" alt="imagen" src="https://github.com/lsst-it/k8s-cookbook/assets/1090349/dc7e82a7-7cfd-4f4c-88bd-94ef06335912">